### DISCUSSION TITLE
Can we create aliases for labels?

### DISCUSSION THREAD

**Discussion by TheBig-O** [TriliumNext/Trilium#3323] [Ideas]
The ability to promote labels for templates is really convenient, but I wonder if there is any way to create an "alias" for them.  As an example, tasks have a label called "doneDate." For readability, it would be great if we could have that label displayed as "Date Completed" or something else appropriate to the situation. There are a number of examples in my workflow where I have abbreviations for my fields but few would be able to decipher them. Not sure it's worth lot of effort, but if it's an easy thing to add somehow, I'd be all about it.
Thanks!

**Comment by zadam** [comment]
Hi, I created an issue for this: https://github.com/zadam/trilium/issues/3329

It makes sense, but is not a high priority item.

**Reply by TheBig-O** [reply]
Awesome! Thank you!
It's a "nice to have," so I get that it's a lower priority. 
Keep up the great work!

**Reply by TheBig-O** [reply]
While working with Collection Views, I realized it had a model for creating aliases that may be fairly easy to implement for Labels. In their code, there is an option to create "Headers" for the attributes you wish to show. By identifying a header with the name you want shown, you are effectively creating an alias for that column. An example is: `#attribute="tag,header=Tags"`
If Labels had a similar option for identifying what is shown on the page, I think it would solve my original request.
Thanks for considering this!
